libraries/apollo-ast/src/main/kotlin/com/apollographql/apollo3/ast/introspection/IntrospectionSchema.kt

```diff
@@ -147,51 +153,36 @@ data class IntrospectionSchema(
   }
 }
 
-/**
- *
- */
+@OptIn(ExperimentalSerializationApi::class)
+private val json: Json by lazy {
+  Json {
+    ignoreUnknownKeys = true
```

Comment:
Curious why this is needed. We might have to tweak the introspection query if we're fetching too many fields

Reply:
It appears that certain fields are always fetched but are used only in some of the type kinds, e.g. `inputFields` only relevant for `InputObject`. 
(Tangent: I never thought about this before, but how come the introspection schema is not "strongly typed"? 😅)